src/release/triage-procedure.md

```diff
@@ -247,9 +247,44 @@ future work.
 
 ## Issue triage
 
-Issue triage is mostly much simpler. After finishing PR triage, go to the [list
-of untagged issues] and add tags as you see fit. The following categories
-should, ideally, be assigned to each issue:
+### Incorrectly Filed Issues
+
+Frequently, issues are filed against the wrong repository. In that case, you can
+use GitHub's "Transfer issue" function to move it to the right place. When you
+do that, please also leave a comment explaining why you moved it.
+
+Some issues might not belong on GitHub at all, in which case they may be closed
+with an appropriate comment pointing to the right place.
+
+Issues in [`rust-lang/rust`] should be closed in the following cases:
+
+* **Language Feature Requests** that might require non-trivial design effort
+  should closed in favor of the [RFC process] or a discussion on [IRLO].
```

Comment:
Perhaps we should direct folks to https://lang-team.rust-lang.org/proposing_a_project.html instead